locked-nft/transactions/unlock_nft_with_authorized_deposit.cdc

```diff
@@ -0,0 +1,65 @@
+import NonFungibleToken from "NonFungibleToken"
+import ExampleNFT from "ExampleNFT"
+import NFTLocker from "NFTLocker"
+import Escrow from "Escrow"
+
+/// This transaction unlocks the NFT with provided ID and adds to an escrow leaderboard, unlocking them if necessary.
+///
+transaction(leaderboardName: String, nftID: UInt64) {
```

Comment:
User transaction to unlock a locked NFT and directly add into escrow leaderboard